jetson/nav/rover.hpp

```diff
@@ -113,14 +106,16 @@ class Rover
 
         Odometry& odometry();
 
-        Target& target();
+        Target& target1();
 
         Target& target2();
 
         RadioSignalStrength& radio();
 
         unsigned getPathTargets();
 
+        RadioSignalStrength mSignal;
```

Comment:
needed?

Reply:
good call, wanted to keep it in but in emergencies we can probs just revert the commit/copy back in code